schema_salad/codegen.py

```diff
@@ -46,7 +47,15 @@ def codegen(
             )
         else:
             dest = sys.stdout
-        gen = PythonCodeGen(dest, copyright=copyright)
+
+        if parser_info:
+            info: Optional[str] = parser_info
+        elif package:
+            info = package
+        else:
+            info = None
```

Comment:
Can you implement the equivalent of https://github.com/common-workflow-language/schema_salad/blob/5dd637fe792f12c985b5d0056b2494ad3220c107/schema_salad/java_codegen.py#L142 instead of `None`?

Reply:
Sure.
Can we move the logic for `self.package` in `java_codegen.py` to `schema_salad.codegen`?
It will prevent code duplication.

Reply:
Sure